taxonomies/countries.txt

```diff
@@ -3105,6 +3105,7 @@ be: Аўстрыя
 be-tarask:Аўстрыя
 be-x-old:Аўстрыя
 bg: Австрия
+regional_entity:en: european-union
```

Comment:
You should add this property after all languages. Together with the others:

> country_code_2:en: AT
> country_code_3:en: AUT
> language_codes:en: de
> osm_relation:en: 16239
> wikidata:en: Q40

And it is by alphebetical order. That would be something like

> country_code_2:en: AT
> country_code_3:en: AUT
> language_codes:en: de
> osm_relation:en: 16239
> regional_entity:en: european-union
> wikidata:en: Q40

Reply:
Note also that Stephane mentioned about having a boolean: https://github.com/openfoodfacts/openfoodfacts-server/issues/12247#issuecomment-3199642984